impl/src/main/java/org/jboss/weld/logging/BootstrapLogger.java

```diff
@@ -342,4 +342,7 @@
     @Message(id = 182, value= "Provided implementation of org.jboss.weld.serialization.spi.ProxyServices ({0}) does not support class defining. This functionality is required in order to be able to define proxy classes.", format = Format.MESSAGE_FORMAT)
     IllegalStateException proxyServicesWithoutClassDefining(Object services);
 
+    @Message(id = 183, value= "Multiple different @Priority values derived from stereotype annotations for annotated type - {0}", format = Format.MESSAGE_FORMAT)
+    DeploymentException multiplePriorityValuesDeclared(Object annotatedType);
```

Comment:
It should be a definition error, not a deployment problem, I think?

Reply:
(I'm not exactly sure what's the difference. Feel free to ignore :rofl:)

Reply:
Good point, thanks!
Fixed.

Reply:
A definition error may be usually detected at development time, e.g. by tooling. Deployment problems are detected at initialization time, i.e. during bootstrap.